src/EnergyPlus/ThermalComfort.cc

```diff
@@ -2219,36 +2222,28 @@ namespace ThermalComfort {
         // within a space.  Future additions might include the effect of direct
         // solar energy on occupants.
 
-        // Return value
-        Real64 CalcRadTemp;
-
-        // Locals
-        Real64 SurfaceTemp;
-
-        // FUNCTION PARAMETER DEFINITIONS:
-        Real64 constexpr AreaEff(1.8);                    // Effective area of a "standard" person in meters squared
-        Real64 constexpr StefanBoltzmannConst(5.6697e-8); // Stefan-Boltzmann constant in W/(m2*K4)
-
-        // FUNCTION LOCAL VARIABLE DECLARATIONS:
-        Real64 ZoneRadTemp;
+        Real64 CalcRadTemp = 0.0;
+        Real64 constexpr AreaEff = 1.8;                    // Effective area of a "standard" person in meters squared
+        Real64 constexpr StefanBoltzmannConst = 5.6697e-8; // Stefan-Boltzmann constant in W/(m2*K4)
 
-        switch (state.dataHeatBal->People(PeopleListNum).MRTCalcType) {
-        case DataHeatBalance::CalcMRT::ZoneAveraged: {
-            state.dataThermalComforts->RadTemp = state.dataHeatBal->ZoneMRT(state.dataThermalComforts->ZoneNum);
+        auto &thisPeople = state.dataHeatBal->People(PeopleListNum);
+        switch (thisPeople.MRTCalcType) {
+        case DataHeatBalance::CalcMRT::EnclosureAveraged: {
+            int enclNum = state.dataHeatBal->space(thisPeople.spaceIndex).radiantEnclosureNum;
+            state.dataThermalComforts->RadTemp = state.dataViewFactor->EnclRadInfo(enclNum).MRT;
```

Comment:
Here's the change from zone to enclosure MRT for people.

Reply:
"So why didn't this PR just change this one line?", he asks as he runs away laughing maniacally.